test/configuration.js

```diff
@@ -167,6 +167,33 @@ describe('configuration', function () {
       chai.config.showDiff = !chai.config.showDiff;
       assert.equal(chai.Assertion.showDiff, chai.config.showDiff);
     });
-    
+  });
+
+  describe('useProxy', function() {
+    var readNoExistentProperty = function() {
+      expect(false).to.be.tue; // typo: tue should be true
+    };
+
+    it('should have default value equal to true', function() {
+      expect(chai.config.useProxy).to.be.true;
+    });
+
+    describe('when true', function() {
+      it('should use proxy unless user\'s environment doesn\'t support', function() {
+        if (typeof Proxy !== 'undefined' || typeof Reflect !== 'undefined') {
```

Comment:
Here we will never test the `else` clause on environments that do support `Proxy`.
What do you guys think about assigning `undefined` to `Proxy` after testing it if the environment supports it, just to see if the code inside the `else` clause will still be valid on every environment? Otherwise, if we test only on environments with `Proxy` support, that condition will never be tested.